src/main/java/org/lightcouch/CouchDbClientBase.java

```diff
@@ -195,13 +195,16 @@ public String getCookie(){
 		 * @param dbName The database name
 		 * @param confirm A confirmation string with the value: <tt>delete database</tt>
 		 */
+		@Deprecated
 		public void deleteDB(String dbName, String confirm) {
-			assertNotEmpty(dbName, "dbName");

```

Comment:
I moved it down to `deleteDB(String dbName)` method, I didn't think there was value for checking if it is empty twice